plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/TestInfluxConnectorTest.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import io.trino.testing.BaseConnectorTest;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.intellij.lang.annotations.Language;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.influxdb.InfluxQueryRunner.createInfluxQueryRunner;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static io.trino.testing.MaterializedResult.resultBuilder;
+import static io.trino.testing.assertions.Assert.assertEquals;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestInfluxConnectorTest
+        extends BaseConnectorTest
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        TestingInfluxServer server = closeAfterClass(new TestingInfluxServer());
+        return createInfluxQueryRunner(server, REQUIRED_TPCH_TABLES);
+    }
+
+    @SuppressWarnings("DuplicateBranchesInSwitch")
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        return switch (connectorBehavior) {
+            case SUPPORTS_CREATE_SCHEMA -> false;
+            case SUPPORTS_CREATE_TABLE, SUPPORTS_RENAME_TABLE -> false;
+            case SUPPORTS_ADD_COLUMN, SUPPORTS_RENAME_COLUMN -> false;
+            case SUPPORTS_COMMENT_ON_TABLE, SUPPORTS_COMMENT_ON_COLUMN -> false;
+            case SUPPORTS_INSERT, SUPPORTS_UPDATE, SUPPORTS_DELETE -> false;
+            case SUPPORTS_ARRAY -> false;
+            default -> super.hasBehavior(connectorBehavior);
+        };
+    }
+
+    @Test
+    @Override
+    public void testShowCreateTable()
```

Comment:
Please leave comment why this is overridden.

Reply:
"time" field is additionally in query result and the column is ordered by name naturally.

Reply:
I knew. I mean leaving code comments.